python/taichi/misc/gui.py

```diff
@@ -228,14 +241,30 @@ def get_events(self, *filter):
             if filter is None or filter.match(e):
                 yield e
 
+    def map_event(self, event):
+        e = GUI.Event()
+
+        e.type = event.type  #int(event.type)
+        e.key = event.key
+
+        if e.type != GUI.WHEEL:
+            e.pos = event.pos()
+        else:
+            dt = event.delta()
+            if dt[0] != 0:
+                e.key = GUI.WHEELLEFT if dt[0] > 0 else GUI.WHEELRIGHT
```

Comment:
I think we could simply:
e.key = WHEEL
e.type = MOTION
? Since dt could already distinguish the direction. We can simply use WHEEL as key.
This can make user loop easy to implement.

Reply:
Well , I had decided to make this wrapping under the API , cause I was not sure whether users do want to make distinction by themselves.
Indeed , this might lead to more complexity and little confusion between WHEEL and WHEELLEFT ,which one is a type and another is a key .

Now we reach the consensus . Let's simplify it .